iree/test/e2e/xla_ops/BUILD

```diff
@@ -93,75 +93,74 @@ iree_check_single_backend_test_suite(
-#             "iota.mlir",
-#             "log.mlir",
-#             "log_plus_one.mlir",
-#             "maximum.mlir",
-#             "minimum.mlir",
-#             "multiply.mlir",
-#             "negate.mlir",
-#             "pad.mlir",
-#             "reduce.mlir",
-#             "reduce_window.mlir",
-#             "remainder.mlir",
-#             "reshape.mlir",
-#             "reverse.mlir",
-#             "rsqrt.mlir",
-#             "scatter.mlir",
-#             "scatter_dynamic.mlir",
-#             "select.mlir",
-#             "sine.mlir",
-#             "slice.mlir",
-#             "sort.mlir",
-#             "sqrt.mlir",
-#             "subtract.mlir",
-#             "tanh.mlir",
-#             "torch_index_select.mlir",
-#             "transpose.mlir",
-#             "while.mlir",
-#         ],
-#         include = ["*.mlir"],
-#         exclude = [
-#             "round.mlir",
-#         ],
-#     ),
-#     compiler_flags = [
-#         "-iree-input-type=mhlo",
-#         "-iree-llvm-link-embedded=true",
-#     ],
-#     driver = "dylib",
-#     target_backend = "dylib-llvm-aot",
-# )
+iree_check_single_backend_test_suite(
+    name = "check_dylib_embedded-llvm-aot_dylib",
+    srcs = enforce_glob(
+        # keep sorted
+        [
+            "abs.mlir",
+            "add.mlir",
+            "batch_norm_inference.mlir",
+            "broadcast.mlir",
+            "broadcast_add.mlir",
+            "broadcast_in_dim.mlir",
+            "clamp.mlir",
+            "compare.mlir",
+            "concatenate.mlir",
+            "constant.mlir",
+            "convert.mlir",
+            "convolution.mlir",
+            "cosine.mlir",
+            "divide.mlir",
+            "dot.mlir",
+            "dot_general.mlir",
+            "dynamic_slice.mlir",
+            "dynamic_update_slice.mlir",
+            "exponential.mlir",
+            "exponential_minus_one.mlir",
+            "fft.mlir",
+            "finite.mlir",
+            "floor.mlir",
+            "gather.mlir",
+            "iota.mlir",
+            "log.mlir",
+            "log_plus_one.mlir",
+            "maximum.mlir",
+            "minimum.mlir",
+            "multiply.mlir",
+            "negate.mlir",
+            "pad.mlir",
+            "reduce.mlir",
+            "reduce_window.mlir",
+            "remainder.mlir",
+            "reshape.mlir",
+            "reverse.mlir",
+            "rsqrt.mlir",
+            "scatter.mlir",
+            "scatter_dynamic.mlir",
+            "select.mlir",
+            "sine.mlir",
+            "slice.mlir",
+            "sort.mlir",
+            "sqrt.mlir",
+            "subtract.mlir",
+            "tanh.mlir",
+            "torch_index_select.mlir",
+            "transpose.mlir",
+            "while.mlir",
+        ],
+        include = ["*.mlir"],
+        exclude = [
+            "round.mlir",
+        ],
+    ),
+    compiler_flags = [
+        "-iree-input-type=mhlo",
+        "-iree-llvm-link-embedded=true",
+    ],
```

Comment:
```suggestion
    data = ["@llvm-project//ld.lld"],
    compiler_flags = [
        "-iree-input-type=mhlo",
        "-iree-llvm-link-embedded=true",
        "-iree-llvm-embedded-linker-path=$(location @llvm-project//ld.lld)",
    ],
```

Reply:
Yeah, thanks. I forgot about this. Is it enough to add that to `iree-opt`, `iree-translate` and `iree-run-mlir`? Thats what I did the last time.

Reply:
Actually I added the dependency on `gen-lld` in `llvm-project/lld`. That seems to be the portable version?